guides/common/modules/proc_generating-metadata-and-listing-files-for-syncable-exports.adoc

```diff
@@ -0,0 +1,26 @@
+[id="generating-metadata-and-listing-files-for-syncable-exports_{context}"]
+= Generating metadata and listing files for syncable exports
+
+In most cases, when exporting content in a syncable format using root privileges and the standard commands, metadata and listing files are generated automatically. However, you must manually trigger the generation of these files in the following circumstances:
```

Comment:
```suggestion
When exporting content in a syncable format, metadata and listing files are generated automatically.
However, you must manually trigger the generation of these files in the following circumstances:
```

* Please use one sentence per line.
* I suggest to drop "in most cases" because you have a list of all other cases below.
* I'd shorten it because we have to assume that users follow the docs.